pymeasure/instruments/common_base.py

```diff
@@ -505,8 +536,38 @@ def measurement(get_command, docs, values=(), map_values=None,
         :param check_get_errors: Toggles checking errors after getting
         :param dynamic: Specify whether the property parameters are meant to be changed in
             instances or subclasses. See :meth:`control` for an usage example.
+        :param separator: A separator character to split the string into a list.
+        :param cast: A type to cast the result.
+        :param preprocess_reply: optional callable used to preprocess values
+            received from the instrument. The callable returns the processed
+            string.
+        :param maxsplit: At most `maxsplit` splits are done. -1 (default) indicates no limit.
+        :param dict v_kwargs: Further keyword arguments for :meth:`values`.
         :param \\**kwargs: Keyword arguments for :meth:`values`.
+
+            ..deprecated:: 0.12
+                Use `v_kwargs` dictionary parameter instead.
         """
+        if kwargs:
```

Comment:
The handling of kwargs I suggested above should enable you here to avoid the duplicated code.